pinot-plugins/pinot-input-format/pinot-csv/src/test/java/org/apache/pinot/plugin/inputformat/csv/CSVRecordReaderTest.java

```diff
@@ -538,8 +538,9 @@ private List<GenericRow> readCSVRecords(File dataFile, CSVRecordReaderConfig rea
 
     try (CSVRecordReader recordReader = new CSVRecordReader()) {
       recordReader.init(dataFile, null, readerConfig);
+      GenericRow reuse = new GenericRow();
       while (recordReader.hasNext()) {
-        GenericRow genericRow = recordReader.next();
+        GenericRow genericRow = recordReader.next(reuse);
```

Comment:
It would be great if we test both `next()` and `next(reuse)`. Is it already the case?

Reply:
addressed – added coverage for both.